python/dgl/homophily.py

```diff
@@ -0,0 +1,57 @@
+"""Utils for tacking graph homophily and heterophily"""
+from . import backend as F, function as fn
+
+__all__ = ["node_homophily"]
+
+
+def node_homophily(graph, y):
+    r"""Homophily measure from `Geom-GCN: Geometric Graph Convolutional Networks
```

Comment:
measurement

Reply:
Measure is the term used in this kind of papers: https://arxiv.org/abs/2110.14446